src/constants/parade-coordinates.js

```diff
@@ -0,0 +1,125 @@
+export default [
```

Comment:
hmm, any thoughts on whether we should include some sort of visual representation of these coordinates so that if we have to change anything here it would be easier to review?

Reply:
That would be great. In theory it is possible to generate a Google Maps link with these:

[Link](https://www.google.com/maps/dir/?api=1&origin=51.51616,-0.14223&destination=51.50499,-0.1265&waypoints=51.5161,-0.14221%7C51.51605,-0.14214%7C51.51603,-0.14212%7C51.51596,-0.1421%7C51.51589,-0.14207%7C51.51563,-0.14198%7C51.51538,-0.1419%7C51.51532,-0.14189%7C51.51523,-0.14186%7C51.51511,-0.14182%7C51.5149,-0.14176%7C51.5146,-0.14166%7C51.51445,-0.14161%7C51.51438,-0.14158%7C51.5143,-0.14154%7C51.5142,-0.14148%7C51.51415,-0.14145%7C51.51409,-0.1414%7C51.51404,-0.14133%7C51.51399,-0.14128%7C51.51393,-0.14123%7C51.51387,-0.14118%7C51.51379,-0.14112%7C51.51372,-0.14107%7C51.51361,-0.14097%7C51.51343,-0.14079%7C51.51314,-0.14055%7C51.51278,-0.14022%7C51.51247,-0.13994%7C51.51219,-0.13969%7C51.51212,-0.13962%7C51.51208,-0.13959%7C51.51205,-0.13955%7C51.51179,-0.13933%7C51.51172,-0.13927%7C51.51166,-0.13921%7C51.51162,-0.13917%7C51.51109,-0.1387%7C51.51047,-0.13814%7C51.51043,-0.13811%7C51.51037,-0.13804%7C51.51034,-0.138%7C51.51029,-0.13792%7C51.51025,-0.13784%7C51.5102,-0.13774%7C51.51016,-0.13762%7C51.51012,-0.13753%7C51.51009,-0.13741%7C51.51006,-0.13729%7C51.51002,-0.13712%7C51.51,-0.13698%7C51.50998,-0.13687%7C51.50997,-0.13678%7C51.50996,-0.13664%7C51.50995,-0.13651%7C51.50995,-0.13638%7C51.50995,-0.13632%7C51.50996,-0.1362%7C51.50996,-0.13607%7C51.50997,-0.13601%7C51.50998,-0.13593%7C51.50998,-0.13591%7C51.51,-0.13576%7C51.51003,-0.13564%7C51.51005,-0.13553%7C51.51007,-0.1354%7C51.51011,-0.13526%7C51.51013,-0.13517%7C51.51014,-0.13515%7C51.51014,-0.13514%7C51.51014,-0.13513%7C51.51014,-0.13512%7C51.51013,-0.1351%7C51.50972,-0.13483%7C51.50964,-0.13476%7C51.50953,-0.13465%7C51.50925,-0.13438%7C51.50918,-0.13433%7C51.50858,-0.13381%7C51.50718,-0.13228%7C51.50734,-0.13182%7C51.50764,-0.131%7C51.50765,-0.13097%7C51.50765,-0.13092%7C51.50765,-0.13087%7C51.50765,-0.13082%7C51.50765,-0.13078%7C51.50762,-0.1305%7C51.50761,-0.13032%7C51.50758,-0.13007%7C51.50756,-0.12995%7C51.50746,-0.12905%7C51.50745,-0.12894%7C51.50744,-0.12887%7C51.50743,-0.12872%7C51.5074,-0.12847%7C51.50737,-0.1283%7C51.50734,-0.12821%7C51.50732,-0.12817%7C51.50728,-0.12808%7C51.50727,-0.12807%7C51.50718,-0.12796%7C51.50716,-0.12793%7C51.50713,-0.12775%7C51.5071,-0.12771%7C51.50707,-0.12766%7C51.50702,-0.12761%7C51.50697,-0.12756%7C51.50692,-0.12749%7C51.50665,-0.12738%7C51.50631,-0.12721%7C51.50626,-0.12722%7C51.50623,-0.12721%7C51.5062,-0.12721%7C51.50613,-0.12717%7C51.50598,-0.12709%7C51.50584,-0.12702%7C51.5056,-0.12692%7C51.50534,-0.1268%7C51.50517,-0.12672%7C51.50497,-0.12664)

It seems it cannot handle the amount of waypoints, though.

Reply:
Oh, do you mean just comment the file with a link?
I don't think the number of waypoints was the problem, it was just that the directions were for car as opposed to walking